BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEventBlob.kt

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+// Ref: https://www.reddit.com/r/javascript/comments/10st04/what_is_a_json_blob/
```

Comment:
Should we remove URL references in code? One concern is that the reddit post is removed at a later date.

Reply:
Yeah we can. I wish we could find a better name that feels more intuitive than "blob." I may replace this with a comment to call out that we're enforcing schema at the JSON level.